_src/pattern-library/sass/components/_grid.scss

```diff
@@ -0,0 +1,1125 @@
+        @include span(7);
+    }
+
+    .col-8 {
+        @include span(8);
+    }
+
+    .col-9 {
+        @include span(9);
+    }
+
+    .col-10 {
+        @include span(10);
+    }
+
+    .col-11 {
+        @include span(11);
+    }
+
+    .col-12 {
+        @include span(full);
+    }
+
+    // shifts - before cols
+    .pre-1 {
+        @include pre(1);
+    }
+
+    .pre-2 {
+        @include pre(2);
+    }
+
+    .pre-3 {
+        @include pre(3);
+    }
+
+    .pre-4 {
+        @include pre(4);
+    }
+
+    .pre-5 {
+        @include pre(5);
+    }
+
+    .pre-6 {
+        @include pre(6);
+    }
+
+    .pre-7 {
+        @include pre(7);
+    }
+
+    .pre-8 {
+        @include pre(8);
+    }
+
+    .pre-9 {
+        @include pre(9);
+    }
+
+    .pre-10 {
+        @include pre(10);
+    }
+
+    .pre-11 {
+        @include pre(11);
+    }
+
+    // shifts - after col
+    .post-1 {
+        @include post(1);
+    }
+
+    .post-2 {
+        @include post(2);
+    }
+
+    .post-3 {
+        @include post(3);
+    }
+
+    .post-4 {
+        @include post(4);
+    }
+
+    .post-5 {
+        @include post(5);
+    }
+
+    .post-6 {
+        @include post(6);
+    }
+
+    .post-7 {
+        @include post(7);
+    }
+
+    .post-8 {
+        @include post(8);
+    }
+
+    .post-9 {
+        @include post(9);
+    }
+
+    .post-10 {
+        @include post(10);
+    }
+
+    .post-11 {
+        @include post(11);
+    }
+}
+
+// ----------------------------
+// #CANNED + RESPONSIVE
+// ----------------------------
+.grid-manual {
+
+    // CASE: small screen bp
+    .sm-col-1 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(1); };
+    }
+
+    .sm-col-2 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(2); };
+    }
+
+    .sm-col-3 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(3); };
+    }
+
+    .sm-col-4 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(4); };
+    }
+
+    .sm-col-5 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(5); };
+    }
+
+    .sm-col-6 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(6); };
+    }
+
+    .sm-col-7 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(7); };
+    }
+
+    .sm-col-8 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(8); };
+    }
+
+    .sm-col-9 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(9); };
+    }
+
+    .sm-col-10 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(10); };
+    }
+
+    .sm-col-11 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(11); };
+    }
+
+    .sm-col-12 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(full); };
+    }
+
+    // shifts - pre
+    .sm-pre-1 {
```

Comment:
NOTE: This whole set of definitions could be made more efficient through a mixin or function, but that seemed like a clean up/optimization task once the grid gets some use.